bears/configfiles/TOMLBear.py

```diff
@@ -0,0 +1,28 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='tomlv',
+        output_format='regex',
+        output_regex=r"Error in '(?P<filename>.+)': "
```

Comment:
@jayvdb Why would regex-collected filename and command line filename would be different? I didn't get you.